views/update/remove-is-entity-registered-owner.html

```diff
@@ -27,10 +27,10 @@ <h1 class="govuk-heading-xl govuk-!-margin-bottom-3">Is the overseas entity curr
         }) }}
 
         {{ govukRadios({
-          errorMessage: errors.owner_disposed if errors,
+          errorMessage: errors.listed_as_property_owner if errors,
           classes: "govuk-radios",
-          idPrefix: "owner_disposed",
-          name: "owner_disposed",
+          idPrefix: "listed_as_property_owner",
```

Comment:
Owner disposed sounded too much like the previous sold land filter page this would cause serious confusion so we need a name that is clearly different here.